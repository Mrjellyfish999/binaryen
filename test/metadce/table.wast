(module
 (table $table 10 funcref)
 (elem $elem (i32.const 0))

 (func $func (export "test")
  (table.fill $table
   (i32.const 0)
   (ref.func $func)
   (i32.const 0)
  )
 )
)

