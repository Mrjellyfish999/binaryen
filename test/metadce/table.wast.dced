(module
 (type $0 (func))
 (elem declare func $func)
 (export "test" (func $func))
 (func $func (type $0)
  (table.fill $table
   (i32.const 0)
   (ref.func $func)
   (i32.const 0)
  )
 )
)
