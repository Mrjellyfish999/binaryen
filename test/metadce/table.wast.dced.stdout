unused: eseg$elem$2
unused: tag_import
